### ISSUE TITLE
Can the package fuse different latex files into one ?

### ISSUE THREAD

**Issue by MohamedAliRashad** [google-research/arxiv-latex-cleaner#65]
I was looking for a tool that can clean different latex files and merge them into one big `tex` file while remove necessaries.
 Is this what happens here ?

**Comment by jponttuset** [comment]
Hi @MohamedAliRashad,
At the moment, this functionality is not implemented, but happy to incorporate a PR.
Best,

**Comment by MohamedAliRashad** [comment]
@jponttuset 
I actually have no idea how something like this could be done ... based on my research it's a task hard said than done.

**Comment by JeremieMelo** [comment]
Just use latexpand main.tex > merged_main.tex

**Comment by sharmaeklavya2** [comment]
Before I knew about the existence of `arxiv-latex-cleaner`, I wrote my own python script to do this: <https://github.com/sharmaeklavya2/tex-flatten>. The basic idea is that you read main.tex, and recursively replace all occurrences of `\input{X}` by the contents of `X`. You just need to be careful about not considering `\input` commands that are inside comments. You can copy code from my repository if you want.

**Comment by tvercaut** [comment]
I agree this would a useful feature. A workaround is however to use a separate tool for it such as `latexpand` which is listed on CTAN and available through TeXLive and MiKTeX:
https://ctan.org/pkg/latexpand
